tests/tests_disk_errors.yml

```diff
@@ -122,7 +122,9 @@
             storage_pools:
               - name: foo
                 disks: "{{ unused_disks }}"
-                type: partition
+#                type: partition
+                volumes:
+                  - name: bar
```

Comment:
It's implemented, just an incorrect key in the class lookup dict. It's fixed in #43.

Reply:
I will revert this part then